packages/billing/stripe-webhook-handlers/stripe.ts

```diff
@@ -0,0 +1,14 @@
+import Stripe from 'stripe';
+
+let stripe: Stripe;
+export function getStripe() {
+  if (!process.env.STRIPE_WEBHOOK_SECRET) {
```

Comment:
Stripe webhook secret is intended for verifying that the webhook came from Stripe. For talking to Stripe's API you need the api key not the webhook secret. Use STRIPE_API_KEY